opengrok-tools/src/main/python/opengrok_tools/scm/mercurial.py

```diff
@@ -108,12 +108,13 @@ def incoming(self):
                               env_vars=self.env, logger=self.logger)
         cmd.execute()
         self.logger.debug(cmd.getoutputstr())
-        if cmd.getstate() != Command.FINISHED:
+        retcode = cmd.getretcode()
+        if cmd.getstate() != Command.FINISHED or retcode not in [0,1]:
```

Comment:
I don't get this, why `1` is special?

Reply:
This is what Mercurial man page says for the `incoming` command:
```
       Returns 0 if there are incoming changes, 1 otherwise.
```

Reply:
aha, perfect